api/server/config.go

```diff
@@ -6,11 +6,12 @@ import (
 
 // Config is used for amplifier configuration settings
 type Config struct {
-	Port          string
-	EtcdEndpoints []string
+	Port             string
+	EtcdEndpoints    []string
+	ElasticSearchURL string
```

Comment:
Really hate to nitpick like this, but technically it is `Elasticsearch` not `ElasticSearch`. https://www.elastic.co/products/elasticsearch